src/assets/Generator.Shared/CodeGenSchemaMemberAttribute.cs

```diff
@@ -8,11 +8,8 @@ namespace Azure.Core
     [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
     internal class CodeGenSchemaMemberAttribute : Attribute
     {
-        public string SchemaName { get; }
```

Comment:
Why the removal of this property, but not the parameter in the constructor?

Reply:
We never read the value through the property, the way we use roslyn allows us to read the parameter value directly.